conformance/utils/kubernetes/helpers.go

```diff
@@ -82,11 +82,11 @@ func GWCMustHaveAcceptedConditionAny(t *testing.T, c client.Client, timeoutConfi
 func gwcMustBeAccepted(t *testing.T, c client.Client, timeoutConfig config.TimeoutConfig, gwcName, expectedStatus string) string {
 	t.Helper()
 
-	var controllerName string
-	waitErr := wait.PollImmediate(1*time.Second, timeoutConfig.GWCMustBeAccepted, func() (bool, error) {
-		ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
-		defer cancel()
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
```

Comment:
Would you mind please pointing out which linter was tripped that led to this change?

Reply:
`wait.PollImmediate` is deprecated

Reply:
PollUntilContextTimeout already sets a timeout on the context , Ithink the one you set is ignored

Reply:
yes, you're right, fix it now.